examples/react/start-basic-rsc/package.json

```diff
@@ -12,31 +12,38 @@
     "format": "prettier --write '**/*' --ignore-unknown"
   },
   "dependencies": {
+    "@babel/plugin-syntax-jsx": "^7.24.1",
+    "@babel/plugin-syntax-typescript": "^7.24.1",
     "@tanstack/react-router": "^1.40.0",
     "@tanstack/router-devtools": "^1.40.0",
     "@tanstack/router-vite-plugin": "^1.39.13",
     "@tanstack/start": "^1.40.1",
-    "redaxios": "^0.5.0",
+    "@vinxi/react-server-dom": "^0.0.3",
     "isbot": "^3.7.0",
-    "react": "^18.3.1",
-    "react-dom": "^18.3.1",
+    "react": "0.0.0-experimental-035a41c4e-20230704",
```

Comment:
For anyone looking at these examples, whats the difference in us using the `0.0.0-experimental-...` version instead of `19.0.0-rc-...`?

Reply:
One works with Vinxi, the other one doesn't. This will change soon.